core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/saved-workflow-section.component.ts

```diff
@@ -77,6 +77,21 @@ export class SavedWorkflowSectionComponent implements OnInit {
     this.router.navigate(['/workflow/new']).then(null);
   }
 
+  /**
+   * duplicate the current workflow. A new record will appear in frontend
+   * workflow list and backend database.
+   */
+  public onClickDuplicateWorkflow(workflowToDuplicate: Workflow): void {
+    this.workflowPersistService.createWorkflow(workflowToDuplicate.name + '_copy', Date.now(), Date.now(), workflowToDuplicate.content)
```

Comment:
timestamps should be removed. they are generated by db.

Reply:
Resolved, will be updated in the next commit